en/docs/quick-starts/fb-as-federated-authenticator-sample.md

```diff
@@ -0,0 +1,136 @@
+# Log into a Sample Application using Facebook
+
+This page guides you through using Facebook as a federated authenticator and logging into an application. Here, we use a **sample application** called Travelocity. 
```

Comment:
Shall we say: 

This page guides you through using Facebook as a federated authenticator and logging into a sample application using Facebook user credentials.